src/js/components/AbstractTabsSelection.js

```diff
@@ -0,0 +1,34 @@
+import Control from './Control';
+import Message from '../constant/Message';
+export default class AbstractTabsSelection extends Control {
+  _setDisabledItem(value, isDisabled) {
+    if (!this._getState().items) {
+      return;
+    }
+    const newItems = [...this._getState().items];
+    newItems.forEach((item, i) => {
+      if (item.label === value) {
```

Comment:
I will fix

Reply:
I think you're misunderstanding the props. The "label" prop is used to display to user, the "value" prop is what used to identify the tab (you can think of it as a "key"). For example a "label" will be "User Information" and a "value" should be "userInfo".

Reply:
Thank you for your opinion.
Should the tab name is "value" and Tab's contents is "contents" ?

Reply:
The "contents" is rendered by the "tabPanel" as in your code. To avoid misunderstanding I think you should change the prop "value" to "tabID" or "tabCode".